packages/salesforcedx-vscode-core/src/commands/forceOrgCreate.ts

```diff
@@ -49,17 +49,21 @@ export class ForceOrgCreateExecutor extends SfdxCommandletExecutor<
 
 export class AliasGatherer implements ParametersGatherer<Alias> {
   public async gather(): Promise<CancelResponse | ContinueResponse<Alias>> {
+    let defaultAlias = DEFAULT_ALIAS;
+    if (vscode.workspace.rootPath) {
```

Comment:
`rootPath` is deprecated in favor of `workspaceFolder`. We are currently using `workspaceFolder[0]` since we haven't done work on supporting multiple workspaces.